### PR TITLE
Use https for all links

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi all

This is a followup to https://github.com/gtk-rs/gtk/pull/1054

Cheers,
Stefan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ Alongside GLib bindings, this crate also includes the infrastructure to
 subclass `GObject` for extending libraries like GTK+. A code example can
 be found in the documentation of this crate in the `subclass` module.
 
-- [Gtk-rs project site](http://gtk-rs.org/)
+- [Gtk-rs project site](https://gtk-rs.org/)
 
-- [Online documentation](http://gtk-rs.org/docs/)
+- [Online documentation](https://gtk-rs.org/docs/)
```
Comment (GuillaumeGomez) [inline README.md:11]: Actually, the good link is this one: https://gtk-rs.org/docs-src/

Comment (dns2utf8) [reply README.md:11]: Ah, there are a lot of links pointing to the old folder

Comment (sdroege) [reply README.md]: @GuillaumeGomez Can we maybe have `docs` forward to `docs-src`? Lots of search results with Google also point to the wrong location.

Comment (GuillaumeGomez) [reply README.md]: It's worth a try at least.

Comment (GuillaumeGomez) [reply README.md]: Apparently, from what I can understand, it requires access to wherever the domain name was registered to be able to add a new redirection. So it's up to @gkoz, I can't do anything about it. :-/

**Pull request merged**